settings/fontFamilies.js

```diff
@@ -0,0 +1,3 @@
+export const BASE = '\'Open Sans\', \'Helvetica Neue\', \'Helvetica\', \'Arial\', sans-serif'
```

Comment:
Never mind. The last update of standardjs is now aware of this use case. I will fix 😄